### PR TITLE
Pylint configuration

### PR DESCRIPTION
**Pull request by kzuraw**
**Story / Bug id:** https://github.com/stxnext-csr/volontulo/issues/562
**Reference person:** @krzysztofzuraw
**Description:**

Added `.pylintrc` file, removed unnecessary `pylint disable`, fixing other pylint issues

**New imports / dependencies:**
- `astroid==1.4.8`
- `pylint==1.6.4`
- `pylint-django==0.7.2`
- `pylint-plugin-utils==0.2.4`

**What tests do I need to run to validate this change:**

Run `pylint apps` from main project directory

### CODE REVIEW

**Comment by magul** [comment]
Great, I will review it during weekend.

File: .pylintrc [hunk 1]
```diff
@@ -0,0 +1,211 @@
+[MASTER]
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=CVS,migrations
```
Comment (magul) [inline .pylintrc]: Do we need to ignore `CVS` here?

File: .pylintrc [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+# List of plugins (as comma separated values of python modules names) to load
+load-plugins=pylint_django
+
+[MESSAGES CONTROL]
+
+# Enable the message, report, category or checker with the given id(s). You can
+# either give multiple identifier separated by comma (,) or put this option
+# multiple time.
+disable=W0403,W0232,E1101,C0111,I0011
```
Comment (magul) [inline .pylintrc]: Can we leave here only `disable=E1101,I0011`? Was that deliberated decision to point here also `W0403`, `W0232` and `C0111`?

**Comment by magul** [comment]
Two simple questions, rest looks ok! Great work! :rocket:

**Comment by kzuraw** [comment]
@magul comments adressed

**Pull request merged**